app/rest/runtime/pom.xml

```diff
@@ -175,11 +175,26 @@
             <version>${syndesis-connectors.version}</version>
           </dependency>
           <!-- ==== SQL ==== -->
+          <dependency>
+            <groupId>io.syndesis</groupId>
+            <artifactId>sql-connector</artifactId>
+            <version>${syndesis-connectors.version}</version>
+          </dependency>
+          <dependency>
+            <groupId>io.syndesis</groupId>
+            <artifactId>sql-start-connector</artifactId>
+            <version>${syndesis-connectors.version}</version>
+          </dependency>
           <dependency>
             <groupId>io.syndesis</groupId>
             <artifactId>sql-stored-connector</artifactId>
             <version>${syndesis-connectors.version}</version>
           </dependency>
+          <dependency>
+            <groupId>io.syndesis</groupId>
+            <artifactId>sql-stored-start-connector</artifactId>
+            <version>${syndesis-connectors.version}</version>
```

Comment:
why not `${syndesis.version}` ?

Why is ${syndesis-connectors.version} defined anyway ? Should be removed imo and point to syndesis.version

Reply:
Y I agree, let's open a different issue for that since it's still in a lot of place and I followed suit here to be consistent. https://github.com/syndesisio/syndesis/issues/750